views/login/index.ejs

```diff
@@ -0,0 +1,47 @@
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Chitter</title>
+    <link rel="stylesheet" href="/style.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Open+Sans:700,600"
+      rel="stylesheet"
+      type="text/css"
+    />
+  </head>
+
```

Comment:
You may want to check layout and partials so you don't have to repeat some of these lines; https://raddy.co.uk/blog/nodejs-express-layouts-and-partials/